csrc/preseg_passes/allocation_order_inference.cpp

```diff
@@ -120,7 +145,9 @@ void mapAllocationDomain(
         return mapped_ids.has(it) || it->isReduction();
       });
 
-  auto mapped_id_iter = mapped_ids.begin();
+  // iterate through reverse order, so the entries iterate from slow to fast
```

Comment:
I may have misunderstood the terms. I assume slow means slow varying and therefore major? If so, isn't mapped_ids already major-to-minor because it's filled in the same order as `ref_alloc_domain`. Why reverse order?

Reply:
Sorry about the confusing terms. Let's stick with innermost and outermost then.

i.e. Here we are iterating from outermost to innermost. Hence the reverse.